storage/s3_test.go

```diff
@@ -479,25 +475,25 @@ func TestS3AclFlagOnCopy(t *testing.T) {
 
 				aclVal := val(r.Params, "ACL")
 
-				setExpectedErrs(t, tc.eAcl, aclVal)
+				assertEqual(t, tc.expectedAcl, aclVal)
```

Comment:
please use `go-cmp` if you have struct/interface assertions.

Reply:
great, thanks :)